File: Data Structures & Algorithms/concatenation-of-array/submission-1.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int[] sol = new int[nums.length * 2];

        for(int i = 0; i < nums.length; i++){
            sol[i] = nums[i];
            sol[nums.length + i] = nums[i];
        }

        return sol;
    }
}

File: Data Structures & Algorithms/subsets/submission-1.py
class Solution:
    def subsets(self, nums: List[int]) -> List[List[int]]:
        sol = []

        def helper(index, running_list):
            nonlocal sol
            
            if index >= len(nums):
                sol.append(running_list[::])
                return
            
            running_list.append(nums[index])
            helper(index + 1, running_list)
            running_list.pop()

            helper(index + 1, running_list)
        
        helper(0, [])
        return sol

